index.md

```
@@ -52,47 +52,58 @@ Create a `.env` file in your working directory with the following content. Repla
+# Cache and RedisQ Configuration
+# CACHE_DIR=/app/data/cache
+# REDISQ_URL=https://zkillredisq.stream/listen.php
+# REDISQ_POLL_INTERVAL_MS=1000
+
+# License Manager Configuration
+# LICENSE_MANAGER_URL=https://lm.wanderer.ltd
+
 ```
 
 > **Note:** If you don't have a Discord bot yet, follow our [guide on creating a Discord bot](https://gist.github.com/guarzo/a4d238b932b6a168ad1c5f0375c4a561) or search the web for more information.
 
```

markdownlint-cli2 (0.17.2)

Blank line inside blockquote
null

(MD028, no-blanks-blockquote)